foundation/foundation-upgrade/src/test/java/com/boozallen/aissemble/upgrade/migration/v1_8_0/AiopsReferenceJavaMigrationSteps.java

```diff
@@ -50,7 +50,7 @@ public void theAiopsReferenceJavaMigrationExecutes() {
 
     @Then("the references are updated to com.boozallen.aissemble.data.access")
     public void theReferencesAreUpdatedToComBoozallenAissembleDataAccess() {
-        validatedFile = getTestFile("/v1_8_0/AiopsReferenceJavaMigration/validation/DataAccessRecordValidated.java");
+        validatedFile = getTestFile("v1_8_0/AiopsReferenceJavaMigration/validation/DataAccessRecordValidated.java");
```

Comment:
I: I originally didn't have the logic to strip leading file separators in `getTestFile`, reasoning that a leading slash means an absolute path and should be treated as one.  However, since the parameter is clearly labelled `subPath`, I decided to reverse course on that. Happy to revert these changes if anyone feels strongly.